spring-mvc/src/main/java/com/kodemon/springmvc/controllers/UserController.java

```diff
@@ -110,4 +114,84 @@ public String login(@RequestParam String username, @RequestParam String password
             return "/login";
         }
     }
+
+    /**
+     * Block specified user. Only administrator can block other user and even admin cannot block another admin.
+     * User remain blocked until admin unblocks him.
+     *
+     * @param username Username of user to be blocked
+     * @param r servlet request to get logged user
+     * @param model Data to display
+     * @return JSP page name
+     */
+    @RequestMapping(value = "/blockUser", method = RequestMethod.POST)
+    public String blockUser(@RequestParam String username, ServletRequest r, Model model) {
+        HttpServletRequest request = (HttpServletRequest) r;
```

Comment:
Can't you somehow simplify this method? Because I can see that you are returning always the same String, so that';s the clue that it can be simplified.